src/components/homesections/homesections.js

```diff
@@ -22,6 +22,28 @@ define(['connectionManager', 'cardBuilder', 'appSettings', 'dom', 'apphost', 'la
         }
     }
 
+    function getSectionLimit(collectionType, shape) {
+        let screenWidth = dom.getWindowSize().innerWidth;
+        const screenHeight = dom.getWindowSize().innerHeight;
+
+        if (cardBuilder.isResizable(screenWidth)) {
+            const roundScreenTo = 100;
+            screenWidth = Math.floor(screenWidth / roundScreenTo) * roundScreenTo;
+        }
+
+        if (!shape) {
+            if (collectionType === 'movies' || collectionType === 'books' || collectionType === 'tvshows') {
```

Comment:
collectionType or viewType === 'mixed' too as they're movies and tv shows!

Reply:
Mixed Content do not have a collectionType, nor do they have a viewType.

Also, as far as I can tell, these libraries use the thumb shape in the latest section, not another shape.

In addition, Mixed Content libraries can contain any kind of content, not only TV or Movies.